hongit-api/src/main/kotlin/com/devfloor/hongit/api/commentfavorite/application/CommentFavoriteService.kt

```diff
@@ -15,7 +15,7 @@ class CommentFavoriteService(
     private val commentFavoriteRepository: CommentFavoriteRepository,
 ) {
     fun create(commentId: Long, user: User): Long = commentRepository.findByIdOrNull(commentId)
-        ?.let { commentFavoriteRepository.save(CommentFavorite(it, user)) }?.run { id }
+        ?.let { commentFavoriteRepository.save(CommentFavorite(it, user)) }?.id
```

Comment:
```suggestion
        ?.let { commentFavoriteRepository.save(CommentFavorite(it, user)) }
        ?.id
```

이건 어때? let 뒤에 붙어있으니깐 한눈에 잘 안 들어오는것 같아서!!

Reply:
반영했어!